assets/stylesheets/common/wizard-admin.scss

```diff
@@ -711,3 +712,22 @@
   width: 80px;
   vertical-align: middle;
 }
+
+.btn.btn-pavilion-pro {
+  background: var(--pavilion-primary);
+  color: var(--pavilion-secondary);
+
+  .d-icon {
+    color: var(--pavilion-secondary);
+  }
+
+  &:hover,
+  &:focus {
+    background: darken($pavilionPrimary, 5%);
```

Comment:
@keegangeorge  were there no `var(-- ` existing equivalents?

Reply:
just wondered if it's worth creating a variable in the standard pavilion set to cover these (if they don't exist already)? If not, please go ahead with your existing approach 👍